specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/costmanagement.generatedetailedcostreport.json

```diff
@@ -0,0 +1,373 @@
+            "headers": {
+              "Location": {
+                "description": "The URL to check the result of the asynchronous operation.",
+                "type": "string"
+              },
+              "Azure-Consumption-AsyncOperation": {
+                "description": "The URL to check the status of the asynchronous operation.",
+                "type": "string"
+              },
+              "Azure-AsyncOperation": {
+                "description": "The URL to check the status of the asynchronous operation.",
+                "type": "string"
+              },
+              "Retry-After": {
+                "description": "The amount of delay to use while the status of the operation is checked. The value is expressed in seconds.",
+                "format": "int32",
+                "type": "integer"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/GenerateDetailedCostReportErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.CostManagement/operationResults/{operationId}": {
+      "get": {
+        "tags": [
+          "GenerateDetailedCostReportOperationResults"
+        ],
+        "operationId": "GenerateDetailedCostReportOperationResults_Get",
+        "description": "Get the result of the specified operation. This link is provided in the GenerateDetailedCostReport creation request response header.",
+        "x-ms-examples": {
+          "Get details of the operation result": {
+            "$ref": "./examples/GenerateDetailedCostReportOperationResultsBySubscriptionScope.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "operationId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The target operation Id."
+          },
+          {
+            "$ref": "#/parameters/scopeUsageDetailsParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/GenerateDetailedCostReportOperationResult"
+            }
+          },
+          "202": {
+            "description": "Accepted. The operation is ongoing."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "common-types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.CostManagement/operationStatus/{operationId}": {
```

Comment:
> "/{scope}/pr [](http://example.com/codeflow?start=3&length=13)

are you adding the operationstatus at each individual scope ? This is not recommeneded especially for delete operations since the resource could go away from ARM cache and then the operation status would not longer be accessible.

Reply:
Yes, operationStatus is supported at each individual scope. There is no Delete API for this operation.